src/components/auth/PasswordResetMessage.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import 'react-testing-library/cleanup-after-each';
+import 'jest-dom/extend-expect';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+import { render, cleanup } from 'react-testing-library';
+import PasswordResetMessage from './PasswordResetMessage';
+import UpdatePasswordMessage from './UpdatePasswordMessage';
+
+afterEach(cleanup);
+
```

Comment:
Do you need this? I thought it was handled already in the setup Test file.

Reply:
We do not have that in the setup file. But I removed it and my tests were still passing, so I guess I truly do not need it.